### PR TITLE
fix: fix slf4j pom

### PR DESCRIPTION
**Pull request by zekai427**
* **Please check if the PR fulfills these requirements**

- The commit message follows our guidelines
- Tests for the changes have been added (for bug fixes / features)
- Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: java/openmldb-taskmanager/src/test/scala/com/_4paradigm/openmldb/taskmanager/TestSparkLauncherUtil.scala [hunk 1]
```diff
@@ -25,4 +26,18 @@ class TestSparkLauncherUtil extends FunSuite {
...
+    val mainClass = classOf[DummySparkApp].getName
+
+    val launcher = SparkLauncherUtil.createSparkLauncher(mainClass)
+    launcher.setConf(SparkLauncher.DRIVER_EXTRA_CLASSPATH, System.getProperty("java.class.path"))
+
+    val sparkAppHandle = launcher.startApplication()
+    Thread.sleep(3000)
+
+    // Submit successfully and get final state
+    // TODO: get lost state since of lack of log4j jar, more info in sparkAppHandle.getError
```
Comment (tobegit3hub) [inline java/openmldb-taskmanager/src/test/scala/com/_4paradigm/openmldb/taskmanager/TestSparkLauncherUtil.scala]: Remove the TODO comment

File: java/openmldb-taskmanager/src/test/scala/com/_4paradigm/openmldb/taskmanager/TestSparkLauncherUtil.scala [hunk 2]
```diff
@@ -25,4 +26,18 @@ class TestSparkLauncherUtil extends FunSuite {
...
+    assert(!sparkAppHandle.getError.isPresent)
```
Comment (tobegit3hub) [inline java/openmldb-taskmanager/src/test/scala/com/_4paradigm/openmldb/taskmanager/TestSparkLauncherUtil.scala:39]: Assert the sparkAppHandle.getState == State.FINISHED

File: java/openmldb-taskmanager/src/main/resources/taskmanager.properties [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+# Spark Config
+#spark.master=yarn
+spark.master=local
+
+#spark.yarn.jars=hdfs:///Users/tobe/openmldb_300_jars/*
+spark.yarn.jars=
+
+#spark.sumbimt.path=/Users/spark
+spark.submit.path=
```
Comment (tobegit3hub) [inline java/openmldb-taskmanager/src/main/resources/taskmanager.properties]: It should be `spark.home` instead of `spark.submit.path`.

**Review by tobegit3hub — CHANGES_REQUESTED** [review]

File: java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/SparkLauncherUtil.scala [hunk 4]
```diff
@@ -38,14 +38,17 @@ object SparkLauncherUtil {
       .setAppResource(TaskManagerConfig.BATCHJOB_JAR_PATH)
       .setMainClass(mainClass)
 
+    if (TaskManagerConfig.SPARK_HOME != null) {
```
Comment (tobegit3hub) [inline java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/SparkLauncherUtil.scala]: Check if it is empty string.

File: java/openmldb-taskmanager/src/test/scala/com/_4paradigm/openmldb/taskmanager/TestSparkLauncherUtil.scala [hunk 5]
```diff
@@ -37,8 +37,7 @@ class TestSparkLauncherUtil extends FunSuite {
     Thread.sleep(3000)
 
     // Submit successfully and get final state
-    // TODO: get lost state since of lack of log4j jar, more info in sparkAppHandle.getError
-    assert(sparkAppHandle.getState.isFinal)
+    assert(sparkAppHandle.getState == SparkAppHandle.State.FINISHED)
```
Comment (tobegit3hub) [inline java/openmldb-taskmanager/src/test/scala/com/_4paradigm/openmldb/taskmanager/TestSparkLauncherUtil.scala]: It is expected to be State.LOST now.
We can assert sparkAppHandle.getError is empty.

**Review by tobegit3hub — CHANGES_REQUESTED** [review]

File: java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/SparkLauncherUtil.scala (hunk 4 above)
Comment (zekai427) [reply java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/SparkLauncherUtil.scala]: Resolved~

File: java/openmldb-taskmanager/src/test/scala/com/_4paradigm/openmldb/taskmanager/TestSparkLauncherUtil.scala (hunk 5 above)
Comment (zekai427) [reply java/openmldb-taskmanager/src/test/scala/com/_4paradigm/openmldb/taskmanager/TestSparkLauncherUtil.scala]: Resolved~

File: java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/SparkLauncherUtil.scala [hunk 6]
```diff
@@ -38,7 +38,7 @@ object SparkLauncherUtil {
       .setAppResource(TaskManagerConfig.BATCHJOB_JAR_PATH)
       .setMainClass(mainClass)
 
-    if (TaskManagerConfig.SPARK_HOME != null) {
+    if (!TaskManagerConfig.SPARK_HOME.isEmpty && TaskManagerConfig.SPARK_HOME != null) {
```
Comment (tobegit3hub) [inline java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/SparkLauncherUtil.scala]: check if it is null first.

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by aceforeverd — APPROVED** [review]

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**